modular_zubbers/code/modules/projectiles/ammunition/ballistic/rifle.dm

```diff
@@ -0,0 +1,2 @@
+/obj/item/ammo_casing/strilka310/phasic
+	can_be_printed = FALSE
```

Comment:
Why make this a modular file? isn't this ammo Skyrat?

Reply:
It's tg ammo https://github.com/Bubberstation/Bubberstation/blob/1c9d1ac9a1796f5830e4bdd4ba98f4f43bb25ce1/code/modules/projectiles/ammunition/ballistic/rifle.dm#L23

Reply:
WAIT ARE THESE UNDOCUMENTED SKYRAT CHANGES???????

Reply:
Coming to a pr that was merged over a year ago to complain is crazy, hop off my dick bruh.